Minor cleanup on profile fragment

msfjarvis opened the pull request:
ecc3213 (Harsh Shandilya, 2 minutes ago)
    ProfileFragment: Use EditView.getText().clear()

    This is the recommended method of setting an EditView to blank, setText("")
    is for TextView

41d8769 (Harsh Shandilya, 4 minutes ago)
    ProfileFragment: Fix code duplication

    When the profile_selector Spinner is instantiated, an ArrayList and
    ArrayAdapter are created for it to inflate values from. The code that
    populates the ArrayList was duplicated from the RefreshSpinner method. We
    create and assign the ArrayList and ArrayAdapter as blank entities then
    call the RefreshSpinner method to populate the ArrayList, allowing us to
    get rid of the duplicate code.

 da03e98 (Harsh Shandilya, 12 minutes ago)
    ProfileFragment: Clear profile name after saving

    Having the profile name linger after saving might suggest to some users
    that the operation was unsuccessful, despite the LunchBar telling them
    otherwise. Clear the name out to avoid the situation.

msfjarvis committed:
ProfileFragment: Clear profile name after saving

Having the profile name linger after saving might suggest to some
users that the operation was unsuccessful, despite the LunchBar telling
them otherwise. Clear the name out to avoid the situation.

msfjarvis committed:
ProfileFragment: Fix code duplication

When the profile_selector Spinner is instantiated, an ArrayList
and ArrayAdapter are created for it to inflate values from. The
code that populates the ArrayList was duplicated from the RefreshSpinner
method. We create and assign the ArrayList and ArrayAdapter as blank
entities then call the RefreshSpinner method to populate the ArrayList,
allowing us to get rid of the duplicate code.

nicholaschum commented on app/src/main/java/projekt/substratum/fragments/ProfileFragment.java:
```diff
@@ -257,6 +257,7 @@ public View onCreateView(
                         backupFunction.execute();
                         Log.d(References.SUBSTRATUM_LOG, selectedBackup.toString());
                         dialog.dismiss();
+                        backup_name.setText("");
```
Have you considered using `backup_name.getText().clear()`?

msfjarvis replied:
Read up a bit, apparently it's the recommended way to clear `EditView`s on Android. Relevant commit pushed.

msfjarvis committed:
ProfileFragment: Use EditView.getText().clear()

This is the recommended method of setting an EditView to blank,
setText("") is for TextView
